cachi2/core/package_managers/yarn/resolver.py

```diff
@@ -195,10 +195,12 @@ def get_component(self, package: Package) -> Component:
                 ),
             ) from e
 
+        purl = self._generate_purl_for_package(resolved_package, self._project)
```

Comment:
nitpick: can assign this statement directly in the call below?

Reply:
@slimreaper35 you pushed a new revision, but didn't respond to this comment. I'm okay if we ignore this, but a resolution comment would be nice :).

Reply:
It was just my personal visual preference of putting the assignment of purl on a separate line. There is no other reason behind it, so, I am okay with both options as well.